### DISCUSSION TITLE
clickhouse support

### DISCUSSION THREAD

**Discussion by newcworld** [langfuse/langfuse#2140] [Ideas]
### Describe the feature or potential improvement

Clickhouse is better at analyzing data than Postgress. When the amount of data is large, it has obvious advantages.

**Comment by marcklingen** [comment]
this is work in progress, see https://github.com/orgs/langfuse/discussions/1902

**Reply by newcworld** [reply]
when it will release

**Reply by marcklingen** [reply]
Probably this or next month, let's merge discussion into the mentioned thread above
